tests/e2e/e2e_util_test.go

```diff
@@ -949,6 +959,20 @@ func (s *IntegrationTestSuite) executeDelegate(c *chain, valIdx int, endpoint st
 
 	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, endpoint)
 	s.T().Logf("%s successfully delegated %s to %s", delegatorAddr, amount, valOperAddress)
+
+	// Validate delegation successful
+	s.Require().Eventually(
```

Comment:
My concern with moving this into the execute function is it adds the responsibility of verifying the execution of the cli command...this is fine if we are looking to verify a specific positive outcome but makes the utility function less extensible for other testing scenarios. Personal opinion is to move it back out and keep it single responsibility like the other exec functions.